.github/labeler.yml

```diff
@@ -19,10 +19,9 @@
     - tests/modules/programs/thunderbird/*
     - modules/services/imapnotify*.nix
     - modules/programs/mu.nix
-    - modules/programs/msmtp*.nix
     - modules/programs/lieer.nix
     - modules/services/lieer.nix
-    - modules/programs/msmtp*.nix
+    - modules/programs/msmtp*
```

Comment:
think we should do a similar rule for the rest ? to catch the folders instead of the nix files only

Reply:
I was looking at nixpkgs' labeler and seems like the glob only works for folders, in which case this MR is incorrect. Will look at the action doc when I have time. Else I will hardcode the paths